Makefile

```diff
@@ -31,6 +31,9 @@ install:
 	install -Dm 644 po/fr.mo "${DESTDIR}${PREFIX}/share/locale/fr/LC_MESSAGES/${_pkgname}.mo"
 	rm -f po/fr.mo
 
+	# Generate state
+	install -Dm 666 "res/state" "${DESTDIR}/var/lib/arch-update/state"
```

Comment:
Not necessary in the case that we can use the already implemented `$statedir`

Reply:
https://github.com/Antiz96/arch-update/pull/148/commits/98ec9afad34990ef522201125d96d4d8caab9543